src/main/java/mate/model/Driver.java

```diff
@@ -49,4 +49,9 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id, name, licenseNumber);
     }
+
+    @Override
+    public String toString() {
```

Comment:
Consider adding more information to the `toString()` method, like the driver's ID and license number (what can be sensitive data, so think would you need that or not). This will make it easier to debug and identify drivers. 🚕